content/docs/SUSHI/configuration/_index.md

```diff
@@ -57,7 +57,7 @@ If an author wants SUSHI only to build the FHIR definition files, and _not_ to d
 SUSHI will then run in FSH-Only mode to produce FHIR definition files only.
 
 When attempting to extract information from an **ImplementationGuide** resource, SUSHI assumes the project structure required by the [template-based IG Publisher](https://build.fhir.org/ig/FHIR/ig-guidance/). The following approach is used to find the **ImplementationGuide** resource:
-* Look for `<root>/ig.ini`, where `<root>` is the folder containing the **fsh** subdirectory. If the **ig.ini** file exists, it will have an `ig` property which gives the path to the **ImplementationGuide** resource, so SUSHI will use this path to find the resource.
+* Look for `<root>/ig.ini`, where `<root>` is the folder containing the **input/fsh** subdirectory. If the **ig.ini** file exists, it will have an `ig` property which gives the path to the **ImplementationGuide** resource, so SUSHI will use this path to find the resource.
```

Comment:
It could be confusing to say the root folder contains the `input/fsh` subdirectory, because it actually contains the input folder (which contains the fsh folder).  As suggested [here](https://chat.fhir.org/#narrow/stream/215610-shorthand/topic/Sushi.20Documentation.20Change/near/240902533), I think it might be better as:
> where `<root>` is the folder containing the **input** folder, which itself includes other subfolders such as **fsh**.

In fact, we could probably even leave off the "which itself..." part.